posix.mak

```diff
@@ -387,10 +387,9 @@ ifneq (,$(findstring $(OS),linux))
 	cp -P $(LIBSO) $(INSTALL_DIR)/$(OS)/$(lib_dir)/
 	ln -sf $(notdir $(LIBSO)) $(INSTALL_DIR)/$(OS)/$(lib_dir)/libphobos2.so
 endif
-	mkdir -p $(INSTALL_DIR)/src/phobos/etc
-	mkdir -p $(INSTALL_DIR)/src/phobos/std
-	cp -r std/* $(INSTALL_DIR)/src/phobos/std/
-	cp -r etc/* $(INSTALL_DIR)/src/phobos/etc/
+	mkdir -p $(INSTALL_DIR)/src/phobos
+	find std -type f -name "*.d" | xargs -n1 -i cp -r --parents {} $(INSTALL_DIR)/src/phobos/
```

Comment:
Yes, I know.  The original version did have an `-o -name "*.di"` extra in there, but I removed it when I realized there weren't actually any `.di` files at present.  Anyway, in light of your other comment, this should be rethought anyway ;-)